nested_conditionals.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Nested conditionals</title>
+  <meta charset="utf-8">
+  <script>
+    var name = prompt("what is your name?");
+    if (name=="flintstones"){
```

Comment:
use spaces around conditions, and always use triple = for comparisson without types
```suggestion
    if (name === "flintstones"){
```